### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/dashboard#358]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://18f.gsa.gov/dashboard/:
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change background to #949494.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(1) > li > span
  - `<span class="status">discovery</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.85:1. Recommendation: change background to #767676.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(1) > li > p:nth-child(2)
  - `<p class="status-description">User needs are researched and i...</p>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.37:1. Recommendation: change text colour to #212121.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(2) > li > p:nth-child(3)
  - `<p class="dashboard-overview-numbers">6</p>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #4ba74b.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(3) > li > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(3) > li > p:nth-child(2)
  - `<p class="status-description">The service is improved, then t...</p>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.29:1. Recommendation: change text colour to #1e1e1e.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
  - html > body > section > div:nth-child(2) > section > ul > a:nth-child(4) > li > p:nth-child(3)
  - `<p class="dashboard-overview-numbers">8</p>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(10) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(13) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(16) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(17) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(18) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(19) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(20) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(21) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.48:1. Recommendation: change background to #2c882c.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > section > div:nth-child(2) > section > section:nth-child(4) > div > div:nth-child(22) > div:nth-child(1) > a > h1 > span
  - `<span class="status beta">beta</span>`
## Summary:
- 15 Errors
